### PR TITLE
Add nested describe block for getAverageDay/re-adds test for pos/neg steps

### PR DESCRIPTION
**Pull request by allisonjw**
**What's this PR do?**
- this branch adds a nested describe block to the getAverageDay method with all 3 paths tested.
- it also re-adds the correct test in for positive and negative step trends 

**Where should the reviewer start?**
- look in the activity test and review the nested describe block

**How should this be manually tested?**
- run NPM test to see changes in the activity test

 **What are the relevant tickets?**
- this will close issue #33

### CODE REVIEW

File: test/Activity-test.js [hunk 1]
```diff
@@ -108,16 +108,18 @@ describe('ActivityRepository', () => {
     );
   });
 
-  it('should return the average number of stairs climbed for all users on a given date', () => {
-    expect(activityRepository.getAverageDay('2019/08/25', 'flightsOfStairs')).to.equal(90);
-  });
+  describe('getAverageDay', () => {
```
Comment (trishalanglois) [inline test/Activity-test.js:111]: Nice use of a describe block!

**Review by cammac60 — APPROVED** [review]
Great work, these describe blocks make our tests a lot easier to follow.
